src/components/CallView/Grid/Grid.vue

```diff
@@ -522,6 +547,32 @@ export default {
 				this.currentPage = Math.max(0, this.numberOfPages - 1)
 			}
 		},
+
+		callParticipantModels: {
+			immediate: true,
+			handler(newModels, oldModels) {
+				if (!this.participantsInitialised) {
+					return
+				}
+				// TODO: patch the array with the new models
+				this.patchOrderedVideos(newModels)
```

Comment:
I hope we didn't lose any reactivity on attributes that we need in `displayedVideos` 😇